packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -141,21 +162,12 @@ module.exports = {
         test: /\.json$/,
         loader: 'json'
       },
-      // "file" loader makes sure those assets get served by WebpackDevServer.
-      // When you `import` an asset, you get its (virtual) filename.
-      // In production, they would get copied to the `build` folder.
-      {
-        test: /\.(ico|jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2)(\?.*)?$/,
-        loader: 'file',
-        query: {
-          name: 'static/media/[name].[hash:8].[ext]'
-        }
-      },
-      // "url" loader works just like "file" loader but it also embeds
-      // assets smaller than specified size as data URLs to avoid requests.
+      // Use url loader to load resources needed from outside the appSrc
+      // directory.
       {
-        test: /\.(mp4|webm|wav|mp3|m4a|aac|oga)(\?.*)?$/,
+        test: /\.(eot|otf|svg|ttf|woff|woff2)(\?.*)?$/,
```

Comment:
I can see a similar argument could be made for other file types too. If anything, that could mean we need to make the size threshold smaller. Because handling them separately would mean going back to maintaining a whitelist, I would not make an exception for those file types at least without more data about how real bundle sizes are affected.